src/components/Board.js

```diff
@@ -0,0 +1,17 @@
+import { Button } from "react-bootstrap";
+import "./Boards.css";
+
+const Board = ({ id, title, owner, status, cards, selected, onSelect }) => {
```

Comment:
There are many props being passed into here, but it looks like only the `title`, `id`, and `onSelect` are used. Can the other listed prop be removed?